packages/syft/src/syft/store/mongo_document_store.py

```diff
@@ -291,14 +291,25 @@ def _set(
                 f"The fields that should be unique are {keys}."
             )
         else:
+            # TODO: h4x fix ?????? how is this supposed to work? This returns if
```

Comment:
Yeah we were not throwing error because ignore duplicates is set to True, but we're not overwriting the data. +1 to override the data in case ignore duplicates is True during .set.